mdjs-layout/load-all.js

```diff
@@ -0,0 +1,5 @@
+import 'https://cdn.jsdelivr.net/npm/modern-normalize@1.1.0/modern-normalize.min.css';
+import 'https://cdn.jsdelivr.net/npm/prism-themes@1.9.0/themes/prism-vsc-dark-plus.min.css';
+import './typography.css';
+import './misc.css';
```

Comment:
`misc.css` is not a good name and not a good approach, but alternatives are worse

this file has 3 things:
- logo size (because slotted content can't be style deeper than 1 level)
- styles for MDJS stories/demos/code blocks (except code highlighting)
- workaround to prevent style specificity issue between Prism theme and Tailwind Typography

this style could be split in 3 files instead, but then it will be harder to preserve correct order of the selectors, because it's actually important

another reasons: people who want to use `misc.css` will likely want to use either everything from it or nothing, so it's not useful to actually split

but maintenance and readability wise I really dislike this file and still trying to justify for myself the split :)

Reply:
If the order is important for cascade, it should be in the same CSS file indeed to prevent human error, I don't have any suggestions for a better name 😅